core/src/main/java/org/elasticsearch/common/network/Cidrs.java

```diff
@@ -113,4 +113,9 @@ static String octetsToCIDR(int[] octets, int networkMask) {
         assert octets.length == 4;
         return octetsToString(octets) + "/" + networkMask;
     }
+
+    public static String createCIDR(long ipAddress, int networkMask) {
+        long blockSize = 1L << (32 - networkMask);
+        return octetsToCIDR(longToOctets(ipAddress - (ipAddress & (blockSize - 1))), networkMask);
+    }
```

Comment:
So the reason for this method was so I could take a random IP address and a random networkMask and turn them into an appropriate CIDR. I'm not sure how to do that without either having this method or recreating a lot of the logic in this class in the test?